### PR TITLE
LOC CHECKIN | dotnet/roslyn master | 20190412

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf [hunk 1]
```diff
@@ -109,12 +89,7 @@
       </trans-unit>
       <trans-unit id="ERR_DuplicateExplicitImpl">
         <source>'{0}' is explicitly implemented more than once.</source>
-        <target state="translated">"{0}" ist mehrfach explizit implementiert.</target>
-        <note />
-      </trans-unit>
-      <trans-unit id="ERR_DuplicatePropertyReadOnlyMods">
-        <source>Cannot specify 'readonly' modifiers on both accessors of property or indexer '{0}'. Instead, put a 'readonly' modifier on the property itself.</source>
-        <target state="new">Cannot specify 'readonly' modifiers on both accessors of property or indexer '{0}'. Instead, put a 'readonly' modifier on the property itself.</target>
+        <target state="translated">"{0}" ist explizit mehrfach implementiert.</target>
```
Comment (jhinder) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf]: "mehrfach explizit"

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf [hunk 2]
```diff
@@ -1294,12 +1234,12 @@
       </trans-unit>
       <trans-unit id="WRN_DefaultExpressionMayIntroduceNullT">
         <source>A default expression introduces a null value when '{0}' is a non-nullable reference type.</source>
-        <target state="new">A default expression introduces a null value when '{0}' is a non-nullable reference type.</target>
+        <target state="translated">Ein Standardausdruck führt einen NULL-Wert ein, wenn "{0}" ein Verweistyp ist, der keine NULL-Werte zulässt.</target>
```
Comment (jhinder) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf]: default is a C# keyword in this context. See also 5 lines below.

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf [hunk 3]
```diff
@@ -2644,7 +2564,7 @@
       </trans-unit>
       <trans-unit id="ERR_IllegalStatement">
         <source>Only assignment, call, increment, decrement, await, and new object expressions can be used as a statement</source>
-        <target state="translated">Nur assignment-, call-, increment-, decrement-, await- und new-Objektausdrücke können als Anweisung verwendet werden.</target>
+        <target state="translated">Nur assignment-, call-, increment-, decrement- und new object-Ausdrücke können als Anweisung verwendet werden.</target>
```
Comment (jhinder) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf]: "await" got lost again.

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf [hunk 4]
```diff
@@ -8620,7 +8540,7 @@ Um die Warnung zu beheben, können Sie stattdessen /reference verwenden (Einbett
       </trans-unit>
       <trans-unit id="ERR_AutoPropertyInitializerInInterface">
         <source>Instance auto-implemented properties inside interfaces cannot have initializers.</source>
-        <target state="needs-review-translation">Automatisch implementierte Eigenschaften in Schnittstellen können keine Initialisierer aufweisen.</target>
+        <target state="translated">Von der Instanz automatisch implementierte Eigenschaften in Schnittstellen können keine Initialisierer aufweisen.</target>
```
Comment (jhinder) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf]: "Automatisch implementierte Instanz-Eigenschaften"?

Comment (janbaltus) [reply src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf]: @jhinder we implemented your suggestions.

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf [hunk 5]
```diff
@@ -82,19 +67,14 @@
         <target state="translated">Der {1}-Typparameter enthält die Einschränkung "unmanaged". "{1}" kann daher nicht als Einschränkung für "{0}" verwendet werden.</target>
         <note />
       </trans-unit>
-      <trans-unit id="ERR_ConstantPatternVsOpenType">
```
Comment (RikkiGibson) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf]: Is it expected that these untranslated resources are removed?

Comment (janbaltus) [reply src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf]: The delivered xlf were loclalized based on 4/1 version of xlf file.
meanwhile there were several updates of xlf.
Can you merge or re-generate xlf after check-in?

Comment (tmeschter) [reply src/Compilers/CSharp/Portable/xlf/CSharpResources.de.xlf]: I'm working on fixing things up now.

**Comment by tmeschter** [comment]
This is ready to go in. Tagging @jinujoseph and @jaredpar for approval.

**Pull request merged**